### DISCUSSION TITLE
Freezes for a second and then goes fast

### DISCUSSION THREAD

**Discussion by 320kg** [ppy/osu#27647] [Q&A] [answered]
So, I have the broke ass Galaxy A13, and I used to play like some 2023 pre-logo-swap version (forgot which) at like stable-ish 40 FPS, but in this version whenever I start it freezes (I can't see the first part of the map), then when it unfreezes it goes insanely fast until I die. On smaller/easier maps I can actually catch up and play, but it occasionally slows/speeds in the middle of the game for some reason, making for an unpleasing experience.

Also, I recall an exit button to be there, which is very convenient because it's hard to do the exit gesture, and it does the pretty voice that says "See you next time...", but now it just vanished. It's nowhere to be found. Literally. If I connect a keyboard, the ESC key used to exit but now it does nothing.

**Comment by peppy** [comment, marked as answer]
hard to help sorry. try again at a later point in development.
